src/Fixer/DoctrineAnnotation/DoctrineAnnotationIndentationFixer.php

```diff
@@ -108,14 +118,16 @@ private function getLineBracesDelta(Tokens $tokens, $index)
     /**
      * @param Tokens $tokens
      * @param int    $newLineTokenIndex
+     * @param array  $annotationPositions
```

Comment:
yep.
sadly, it's not easy to document non-structured elements like array of pairs :( please provide that info in param description